frontera/contrib/scrapy/converters.py

```diff
@@ -8,15 +8,24 @@
 
 class RequestConverter(BaseRequestConverter):
     """Converts between frontera and Scrapy request objects"""
-    @classmethod
-    def to_frontier(cls, request):
+    def __init__(self, spider):
+        self.spider = spider
+
+    def to_frontier(self, request):
         """request: Scrapy > Frontier"""
         if isinstance(request.cookies, dict):
             cookies = request.cookies
         else:
             cookies = dict(sum([d.items() for d in request.cookies], []))
+        cb = request.callback
+        if callable(cb):
+            cb = _find_method(self.spider, cb)
+        eb = request.errback
+        if callable(eb):
+            eb = _find_method(self.spider, eb)
         meta = {
-            'scrapy_callback': request.callback,
+            'scrapy_callback': cb,
+            'scrapy_errback': eb,
             'origin_is_frontier': True,
         }
         meta.update(request.meta or {})
```

Comment:
We should put a serialized data under a reserved key in this meta. Otherwise users will be able to override internal flags like 'origin_is_frontier' or 'scrapy_callback'.